### PR TITLE
[COMPOSER] Updated scripts, readme files and updated some code

### PR DESCRIPTION
**Pull request by MProG**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+```
+
+## Usage
+
+Creating new app:
+```
+rails_apps_composer new app_name
+```
+
+This is changed rails_apps_composer gem for datarockets team.
```
Comment (AleksSenkou) [inline README.md:22]: @MProG can you please describe both `api` and `fullstack`  recipes in separate README files and paste links for them here

File: README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# datarockets rails-composer
+
+## Install
+
+Install specific_install wich hel install forked gems:
```
Comment (AleksSenkou) [inline README.md]: @MProG `wich hel` :)

File: files/circle.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+dependencies:
+  post:
+    - bin/setup
+test:
```
Comment (AleksSenkou) [inline files/circle.yml]: @MProG can we have empty lines before `test` and `development`?

File: files/gitignore.txt [hunk 4]
```diff
@@ -20,3 +21,4 @@
 /db/schema.rb
 
 /config/puma.rb
+/.env
```
Comment (AleksSenkou) [inline files/gitignore.txt]: @MProG is it works?) I mean passing `.env` file as a folder here?

File: files/circle.yml (hunk 3 above)
Comment (MProG) [reply files/circle.yml]: For wat? This is config very easy and small.

Comment (AleksSenkou) [reply files/circle.yml]: @MProG yeap, but in other `yml` files we do it

File: README.md (hunk 1 above)
Comment (MProG) [reply README.md:22]: @AleksSenkou Maybe in future but now i'll not do this because we i add new points for receiples.

Comment (AleksSenkou) [reply README.md:22]: @MProG what do you think about adding here those things that you've already added to the recipes? Plus in the future we'll just update README with new changes

File: files/gitignore.txt (hunk 4 above)
Comment (MProG) [reply files/gitignore.txt]: @AleksSenkou  
https://github.com/datarockets/rails_apps_composer/issues/43 https://github.com/datarockets/bevital/blob/master/.gitignore

File: files/circle.yml (hunk 3 above)
Comment (MProG) [reply files/circle.yml]: @AleksSenkou If you want i'll do this only for you)\

File: README.md (hunk 1 above)
Comment (MProG) [reply README.md:22]: @AleksSenkou I update some points. And update readme for all changes this is not cool. I add when i finished with logic of gem

Comment (AleksSenkou) [reply README.md:22]: @MProG ok, can you please add some reminder for it(maybe issue)

**Pull request merged**